lib/api/units/validateHeaders.js

```diff
@@ -0,0 +1,67 @@
+const { HeadersJsonExample } = require('../../validators/headers-json-example');
+
+const APIARY_JSON_HEADER_TYPE = 'application/vnd.apiary.http-headers+json';
+
+function normalizeHeaders(headers) {
```

Comment:
Hmm, shouldn't we require from `normalize`? Seems like duplicate code.

Reply:
It's a leftover. Now the normalization of HTML messages happens in `validateMessage`, so all the validation units operate on already normalized data. I will remove this normalization at all.